README.md

```diff
@@ -65,43 +119,21 @@ when processed, produce Markdown content. Here's an example:
 }
 ```
 
-**Important**: when adding or editing return values, be sure to edit both files. Use the following
-links for the reply type. Note: do not use `@reply-type` specifiers; use only the Markdown link.
-
-**Important**: all links below are under construction.
-
-```md
-@simple-string-reply: [Simple string reply](https://valkey.io/topics/protocol#simple-strings)
-@simple-error-reply: [Simple error reply](https://valkey.io/topics/protocol#simple-errors)
-@integer-reply: [Integer reply](https://valkey.io/topics/protocol#integers)
-@bulk-string-reply: [Bulk string reply](https://valkey.io/topics/protocol#bulk-strings)
-@array-reply: [Array reply](https://valkey.io/topics/protocol#arrays)
-@nil-reply: [Nil reply](https://valkey.io/topics/protocol#bulk-strings)
-@null-reply: [Null reply](https://valkey.io/topics/protocol#nulls)
-@boolean-reply: [Boolean reply](https://valkey.io/topics/protocol#booleans)
-@double-reply: [Double reply](https://valkey.io/topics/protocol#doubles)
-@big-number-reply: [Big number reply](https://valkey.io/topics/protocol#big-numbers)
-@bulk-error-reply: [Bulk error reply](https://valkey.io/topics/protocol#bulk-errors)
-@verbatim-string-reply: [Verbatim string reply](https://valkey.io/topics/protocol#verbatim-strings)
-@map-reply: [Map reply](https://valkey.io/topics/protocol#maps)
-@set-reply: [Set reply](https://valkey.io/topics/protocol#sets)
-@push-reply: [Push reply](https://valkey.io/topics/protocol#pushes)
-```
-
-## Styling guidelines
+### Styling guidelines
 
 Please use the following formatting rules (aiming for smaller diffs that are easier to review):
 
-* No need for manual lines wrapping at any specific length,
-  doing so usually means that adding a word creates a cascade effect and changes other lines.
-* Please avoid writing lines that are too long,
-  this makes the diff harder to review when only one word is changed.
-* Start every sentence on a new line.
```

Comment:
I still prefer this style for doc writing, it makes it easy to scan to the next sentence when reviewing.

Reply:
OK, I'll put the guideline back.

Personally, I like the Alt-Q feature in emacs (reflow the current paragraph, works also in code comments). Since I started using it, I can't stop.

Reply:
Yeah, I know there are a bunch of different flow features on line length, but it's hard to get everyone to consistently use them. "One sentence per line" is pretty easy.